### ISSUE TITLE
Cubestore with external bucket support parquet file format

### ISSUE THREAD

**Issue by rongfengliang** [cube-js/cube.js#3051]
**Is your feature request related to a problem? Please describe.**

current cubestore support external bucket witj csv file format . can support with parquet format directly

**Describe the solution you'd like**
we can config bucket file format. maybe we can directly write to cubestore parquet format storage ?

**Comment by paveltiunov** [comment]
Hey @rongfengliang ! Thanks for posting it! Yep. We're considering having more different formats other than CSV to speed up ingestion. Could you please elaborate on your use case? What database do you use and why do you think you'll benefit from using parquet instead of CSV?

**Comment by rongfengliang** [comment]
> Hey @rongfengliang ! Thanks for posting it! Yep. We're considering having more different formats other than CSV to speed up ingestion. Could you please elaborate on your use case? What database do you use and why do you think you'll benefit from using parquet instead of CSV?

we use parquet file as our data lake format so we want cubestore externel bucket can read parquet file directly

**Comment by paveltiunov** [comment]
@rongfengliang What query engine do you use for your data lake?

**Comment by rongfengliang** [comment]
> @rongfengliang What query engine do you use for your data lake?

dremio

**Comment by paveltiunov** [comment]
@rongfengliang Do you know if Dremio supports parquet export? Or do you mean you want Cube Store to access raw parquet files directly from the bucket?

**Comment by rongfengliang** [comment]
> @rongfengliang Do you know if Dremio supports parquet export? Or do you mean you want Cube Store to access raw parquet files directly from the bucket?

dremio can using create table into external storge (like s3. with parquet format) maybe cube store can load this file

**Comment by rascasse83** [comment]
There is quite a lot of overlap functionality wise between Dremio and Cube, as both act as a semantic layer decoupling the presentation layer from the data storage, and enforcing a security model too, at run-time.
Therefore for Cube.js to be considered a possible alternative to Dremio, it would need the ability to connect to data lakes.
Failing that, one could potentially implement a 2-tier semantic layer with Layer 1 being Dremio, federating both RDBMS data and Data Lake files, and Layer 2 being Cube on top of that, but i am not sure whether that's feasible security wise, functionality wise and even practical, as it seems the data would have to go through a lot of hoops and loops to get to the presentation dashboard layer.

**Comment by rongfengliang** [comment]
@rascasse83  Yes,  but cube.js is mainly for bi,dremio for data lake query & semantic layer.  we can use cube.js  dremio driver connect dremio and do some  dashboard
